### ISSUE TITLE
Dealing with bounds size change (e.g. interface orientation change)

### ISSUE THREAD

**Issue by DZamataev** [henrytkirk/HTKDragAndDropCollectionViewLayout#6]
with such setup
```
HTKDragAndDropCollectionViewLayout *flowLayout = (HTKDragAndDropCollectionViewLayout *)self.collectionView.collectionViewLayout;
    CGFloat itemWidth = CGRectGetWidth(self.collectionView.bounds) / 3;
    flowLayout.itemSize = CGSizeMake(itemWidth, itemWidth);
    flowLayout.minimumInteritemSpacing = 0;
    flowLayout.lineSpacing = 0;
    flowLayout.sectionInset = UIEdgeInsetsMake(0, 0, 0, 0);
```
after performing interface rotation and thus changing collectionView width from 320 to 568 i expect items to "reflow" and form less rows with 5 items in a row instead of 3.
Is it possible to achieve it manually with the existing API?
Should it be done automatically?

**Comment by henrytkirk** [comment]
Good catch. It's easy fix. Pushing up fix.

**Comment by henrytkirk** [comment]
Fixed in commit 31ceb80e6e392313934bc715cb574199012c0677
